Gemfile

```diff
@@ -30,6 +30,12 @@ gem "rake", "10.1.1", :require => false
 # Hiera has an unbound dependency on json_pure
 # json_pure 2.0.2+ officially requires Ruby >= 2.0, but should have specified that in 2.0
 gem 'json_pure', '~> 1.8', :require => false
+# i18n support (gettext-setup and dependencies)
+gem 'text', '~> 1.3.1', :require => false
+gem 'locale', '~> 2.1.2-2', :require => false
+gem 'gettext', '~> 3.2.2', :require => false
+gem 'fast_gettext', '~> 1.1.0', :require => false
+gem 'gettext-setup', '~> 0.6', :require => false
```

Comment:
We also need to list the required dependencies in [`project_data.yaml`](https://github.com/puppetlabs/puppet/blob/master/ext/project_data.yaml#L19-L22) which is used when building puppet as a gem, and in [`.gemspec`](https://github.com/puppetlabs/puppet/blob/master/.gemspec#L37-L38) which is used when another project (typically a module) tests against a source checkout of puppet (like we do in module CI).